Sources/PackageLoading/Diagnostics.swift

```diff
@@ -56,6 +56,10 @@ extension Basics.Diagnostic {
         .error("system library product \(product) shouldn't have a type and contain only one target")
     }
 
+    static func libraryProductWithExectuableTarget(product: String) -> Self {
+        .error("library product '\(product)' should not contain an executable target")
```

Comment:
Should the message contain the name(s) of the executable targets? Would make it easier for folks to resolve the issue.

Reply:
+1 if possible

Reply:
Good point.

Reply:
I was trying to match the existing style but can add the target names to some of the other messages here that don't have it.